uat/testing-features/src/main/java/com/aws/greengrass/IoTShadowLifecycle.java

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Created shadow resources similar to what we have in the otf repo for other aws resources - https://github.com/aws-greengrass/aws-greengrass-testing/tree/main/aws-greengrass-testing-resources/aws-greengrass-testing-resources-iot/src/main/java/com/aws/greengrass/testing/resources/iot